primaryHandler.go

```diff
@@ -228,8 +225,8 @@ func provideAuthentication(in provideAuthenticationIn) (*alice.Chain, error) {
 		basculehttp.WithEErrorResponseFunc(listener.OnErrorResponse),
 	)
 
```

Comment:
Everything in `provideAuthentication()` should be covered through the bascule provide functions.

Note: This will get rid of the v2 compatible functionality.  Changes should be made in bascule itself, I think.  We should worry about that after this pr.